core/input/input_event.cpp

```diff
@@ -387,7 +387,7 @@ String InputEventKey::to_string() {
 Ref<InputEventKey> InputEventKey::create_reference(uint32_t p_keycode) {
```

Comment:
Shouldn't this take `Key` t oo?

Reply:
This method often accepts unions of keys, so uint32_t is the easiest option. This one isn't exposed either, so it's not important for C# compatibility.